Assets/CodeBase/Entities/player/Player.cs

```diff
@@ -213,8 +213,9 @@ private void OnCollisionEnter2D(Collision2D collision)
         if (collision.gameObject.CompareTag("Floor"))
         {
             //Setting the free-fall velocity to 0 prevents boosted jumps at corners.
-            playerRigidBody.velocity = new Vector2(playerRigidBody.velocity.x, 0.0f);
+            stopMoving();
```

Comment:
Keep in mind that stopMoving() will make the x velocity 0, not the y velocity. Was this your intent?

Reply:
the stopMoving method does the same as the above line that I removed, I don't actually know if this line serves a purpose, but I thought why not use the readable method for it instead

Reply:
stopMoving invokes setXVelocity(0.0f), which invokes the folowing:
playerRigidBody.velocity = new Vector2(newXVelocity, playerRigidBody.velocity.y);
so while the line you removed has the second parameter of Vector2 go to 0.0f, stopMoving will make the first parameter of Vector2 go to 0.0f. Therefore, stopMoving does not perform the same task as the line you removed.

Reply:
yeeee gona revert these